e2e/playwright/folder-rules/src/tests/create-rules.e2e.ts

```diff
@@ -0,0 +1,226 @@
+    const checkInValue = 'check In Value';
+    const actionValue = ' A site which contains sfdc content [sfdc:site] ';
+    const autoDeclareOptionsValue = 'For all major and minor versions [ALL]';
+    const simpleWorkFlow = 'accept reject';
+
+    await personalFiles.navigate({ remoteUrl: `#/nodes/${folderId}/rules` });
+    await nodesPage.toolbar.clickCreateRuleButton();
+    await nodesPage.manageRulesDialog.ruleNameInputLocator.type(randomRuleName);
+
+    await nodesPage.actionsDropdown.selectAction(ActionType.HideRecord, 0);
+    await nodesPage.actionsDropdown.selectAction(ActionType.IncrementCounter, 1);
+    await nodesPage.actionsDropdown.selectAction(ActionType.CheckIn, 2);
+    await nodesPage.actionsDropdown.insertCheckInActionValues(checkInValue, 2);
+    await nodesPage.actionsDropdown.selectAction(ActionType.AddAspect, 3);
+    await nodesPage.actionsDropdown.insertAddAspectActionValues(actionValue, 3);
+    await nodesPage.actionsDropdown.selectAction(ActionType.AutoDeclareOptions, 4);
+    await nodesPage.actionsDropdown.insertAutoDeclareOptionsActionValues(autoDeclareOptionsValue, 4);
+    await nodesPage.actionsDropdown.selectAction(ActionType.SimpleWorkflow, 5);
+    await nodesPage.actionsDropdown.insertSimpleWorkflowActionValues(simpleWorkFlow, 5);
+
+    await nodesPage.manageRulesDialog.createRuleButton.click();
+
+    await expect.soft(nodesPage.manageRules.getGroupsList(randomRuleName)).toBeVisible();
+  });
+
+  test('[XAT-889] Create a rule which runs when items are deleted or leave a folder', async ({ nodesPage, personalFiles }) => {
+    await personalFiles.navigate({ remoteUrl: `#/nodes/${folderId2}/rules` });
+    await nodesPage.toolbar.clickCreateRuleButton();
+    await nodesPage.manageRulesDialog.ruleNameInputLocator.fill(randomRuleName2);
+    await nodesPage.manageRulesDialog.whenCreatedCheckbox.click();
+    await nodesPage.manageRulesDialog.whenDeletedCheckbox.click();
+    await nodesPage.actionsDropdown.selectAction(ActionType.Copy, 0);
+    await nodesPage.manageRulesDialog.destinationFolderButton.click();
+    await nodesPage.contentNodeSelectorDialog.getFolderIcon.click();
+    await nodesPage.contentNodeSelectorDialog.getOptionLocator(username).click();
+    await personalFiles.dataTable.getRowByName(randomFolderName3).click();
+    await personalFiles.page.keyboard.press('Enter');
+    await nodesPage.contentNodeSelectorDialog.actionButton.click();
+    await nodesPage.manageRulesDialog.createRuleButton.click();
+    await expect(nodesPage.manageRules.getGroupsList(randomRuleName2)).toBeVisible();
+
+    await personalFiles.navigate({ remoteUrl: `#/personal-files/${folderId2}` });
+    await personalFiles.dataTable.selectItem(copyFileName);
+    await personalFiles.acaHeader.clickMoreActions();
+    await personalFiles.acaHeader.matMenu.clickMenuItem('Delete');
+    await personalFiles.snackBar.message.waitFor({ state: 'visible' });
```

Comment:
waiting for the snackbar might take a lot of time, depending on the timer settings in the app, you can skip this part probably, as the http request is made already